scripts/mainnet/reset.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -x
+
```

Comment:
Add at least `set -e` to have the script exit if one step fails.

I also usually add `u` and `o pipefail`. I don't remember what the u does though.

So `set -euo pipefail`.